pkg/build/pipelines/cargo/build.yaml

```diff
@@ -46,7 +46,7 @@ pipeline:
       cd "${{inputs.modroot}}"
 
       # Build and install package(s)
-      cargo auditable build "${{inputs.opts}}"
+      cargo auditable build --target-dir target "${{inputs.opts}}"
```

Comment:
does this need to remove the quotes too such as in https://github.com/chainguard-dev/melange/pull/1593/files ?